PhysicsTools/HeppyCore/scripts/heppy_batch.py

```diff
@@ -63,7 +63,7 @@ def batchScriptCERN( jobDir, remoteDir=''):
 do
    ff=`basename $f | cut -d . -f 1`
    #d=`echo $f | cut -d / -f 2`
-   gfal-mkdir -p {srm}
+   gfal-mkdir {srm}
```

Comment:
ping .... can we fix this three-character difference so that we can merge the PR?